charms/openfga-k8s/metadata.yaml

```diff
@@ -37,8 +37,8 @@ description: |
   Community section for third-party SDKs and tools.
 
 peers:
-  openfga-peer:
-    interface: openfga-peer
+  peer:
+    interface: openfga_peer
```

Comment:
To be fair I do think the `-` gets converted to `_` for use in any function signatures.